pkg/3scale/amp/product/3scale_release.go

```diff
@@ -1,5 +1,5 @@
 package product
 
 var (
-	ThreescaleRelease = "2.7"
```

Comment:
why master?

Reply:
master branch deploys nightly images, so `ThreescaleRelease = "2.7"` is not correct. It could also be "coming release". 

Basically this is needed to use default AMP_RELEASE param when deploying templates or via operator. As the tests were setting `--param AMP_RELEASE=master`.